.github/workflows/ci_cd.yml

```diff
@@ -0,0 +1,110 @@
+name: Client & CMS CI/CD
```

Comment:
any github action script could be called "CI/CD" if you try hard enough. Be more descriptive, so it's easily understandable what this particular action achieves

Reply:
Hmm, I could probably brainstorm some github actions which won't be responsible for CI/CD :). My idea was that this workflow will in the end run also tests, linter and deploy so basically all CI/CD stuff and in that case, name makes sense from my point of view. Ofc I can name it `something related to deploy` and use workflow dependency to keep multiple files such as https://github.com/orgs/community/discussions/26238

Reply:
For now workflow does only deploy so I have changed name to respect this fact...